### ISSUE TITLE
add `add_optional_drivers` option

### ISSUE THREAD

**Issue by haraldh** [dracutdevs/dracut#1159]
**Describe the enhancement**
Install a kernel module optionally and don't error, if it does not exist.

See also https://github.com/systemd/mkosi/pull/680

**Comment by johannbg** [comment]
I dont think we should follow the systemd/Mkosi team into the rabbit hole they are into in systemd/mkosi#680 . 

Vendors/users always have to declare if they want to include/exclude drivers.
There is no option/soft/weak dependency etc. in this case.  

If the systemd/Mkosi team wants to implement hackish workarounds for bad and or non-existing vendor ( Debian ) choices and start a quest of reading minds or intentions, then let's just allow them to do just that, in systemd/Mkosi...
